.teamcity/Gradle_Check/configurations/GradleBuildConfigurationDefaults.kt

```diff
@@ -27,6 +27,14 @@ fun shouldBeSkipped(subProject: GradleSubproject, testConfig: TestCoverage): Boo
     return testConfig.os.ignoredSubprojects.contains(subProject.name)
 }
 
+val killAllGradleProcesses = """
+    free -m
+    ps aux | egrep 'Gradle(Daemon|Worker)' | awk '{print ${'$'}2}'
```

Comment:
what is the reason for duplicating this?
if to show and to kill, maybe we could store the results in a variable, something like
```bash
${"PIDS=$(ps aux | egrep 'Gradle(Daemon|Worker)' | awk '{print \$2}')"}
echo PIDS
echo PIDS | xargs kill -9
```

Reply:
Actually, since we only need the process ID, so we could even use `pgrep` here, i.e.:
```bash
pgrep 'Gradle(Daemon|Worker)' | xargs kill -9
```

Reply:
Now I also print the command line args.